src/index.js

```diff
@@ -0,0 +1,162 @@
+    }
+
+    setListToLocal() {
+        localStorage.setItem('lists', JSON.stringify(this.toDoTasksArray));
+    }
+
+    getListFromLocal() {
+        const getList = localStorage.getItem('lists');
+        if (getList) {
+            this.toDoTasksArray = JSON.parse(getList);
+        }
+        this.displayToDo();
+    }
+
+    displayToDo() {
+        this.toDoTasksArray.sort((a, b) => a.index - b.index);
+
+        this.container.innerHTML = this.toDoTasksArray.map((todo) => `
+        <article id=${todo.index} class="article">
+        <div class="linn">
+        <input ${todo.completed ? 'checked' : ''} type="checkbox">
+        <textarea class = "${todo.completed ? 'complete' : ''} text-area-class" rows="1" cols="30">${todo.description}</textarea> 
+        <button id=${todo.index} class="todo-btn" > &#128465;</button></div>
+        <hr class="line-break">
+        </article>`).join('');
+    }
+
+    // updateTodo(e){
+    //   const clicked = e.target.closest('.text-area-class');
+    //   if (!clicked) return;
+    
+    //   const tasks = getListFromLocal();
+    //   const listNum = +clicked.dataset.desc;
+    
+    //   const task = tasks.find((task) => task.index === listNum);
+    //   task.description = clicked.value.trim();
+    
+    //   setListToLocal(tasks);
+    // };
+
+}
+
+const myToDo = new ToDoList([], '.list-item');
+
+document.addEventListener('click', (e) => {
+    interactive(e);
+    const todoId = e.target.parentElement.parentElement.id;
+    if (e.target.checked) {
+        const upDatedToDo = myToDo.getToDoArray().map((todo) => {
+            if (todo.index === parseInt((todoId), 10)) {
+                const newTodo = {...todo };
+                newTodo.completed = true;
+                return newTodo;
+            }
+            return todo;
+        });
+        myToDo.setToDoArray(upDatedToDo);
+    } else {
+        const upDatedToDo = myToDo.getToDoArray().map((todo) => {
+            if (todo.index === parseInt((todoId), 10)) {
+                const newTodo = {...todo };
+                newTodo.completed = false;
+                return newTodo;
+            }
+            return todo;
+        });
+        myToDo.setToDoArray(upDatedToDo);
+    }
+});
+
+const inputField = document.querySelector('.inputField');
+const inputTodo = document.getElementById('input-todo');
+
+inputField.addEventListener('keyup', (e) => {
+    if (e.key === 'Enter') {
+        myToDo.addToDo(inputTodo.value);
+        inputTodo.value = '';
+    }
+});
+// const list1 =document.querySelector('article')
+// list1.addEventListener('click', (e) => {
+//   const input = content.querySelector('text-area-class');
+//   input.removeAttribute('readonly');
+//   input.focus();
+//   input.addEventListener('blur', (e) => {
+//     input.setAttribute('readonly', true);
+//     todo.content = e.target.value;
+//     localStorage.setItem('todos', JSON.stringify(todos));
+//     displayTodo()
+
+//   })
+// })
```

Comment:
Hi [@EyasuTeshome  )

Good job so far! 👏
you are almost there, There are some issues that you still need to work on.

![cca0aa26-c843-47ac-bf97-7733936ba866_text](https://user-images.githubusercontent.com/96333200/215345412-c4a2117e-80c0-481d-80b0-b5ded662519d.gif)


##Highlights

-  Linter result is green ✔️

-  Commit messages are meaningful✔️

-  The pull request has a descriptive title telling you what changes are inside ✔️

-  The pull request has a summary describing introduced changes in more detail ✔️

-  node_modules folder is in the .gitignore ✔️

## Required Changes ♻️

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
 ##

- [ ]  Kindly check the correct configuration for your webpack because when I run `npm start `or `npm run build`, it gives these errors.
- [ ] Kindly also add instructions for how to run the project using webpack in the readme, so the next reviewer can review your add /remove/ edit functionality without any problems.
`npm start`
![review](https://user-images.githubusercontent.com/96333200/215346136-a765a472-ba2a-46df-8783-623535c15913.png)

- [ ]  [OPTIONAL] Commit messages must start with a Captial alphabet for best practice

- Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- please Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.